rust/src/schema.rs

```diff
@@ -168,10 +296,10 @@ impl Field {
 
 fn to_str(unit: &TimeUnit) -> &'static str {
     match unit {
-        TimeUnit::Second => { "s" }
-        TimeUnit::Millisecond => { "ms" }
-        TimeUnit::Microsecond => { "us" }
-        TimeUnit::Nanosecond => { "ns" }
+        TimeUnit::Second => "s",
```

Comment:
Can you implement this as

```rust

use std::string::ToString;

impl ToString for TimeUnit {
  fn to_string(&self) -> {
     ....
  }
}
```

Reply:
Can't do that. The library already has its own version of ToString and not compatible with our requirements.

Reply:
This method should not belong to `Field`, same as a few other methods.

You can implement a trait called `ToLogicalType`?

Reply:
`fn to_str(unit: &TimeUnit)` is an independent function on the top level of `schema.rs`, I moved the other to methods into `ToLogicalType`